### ISSUE TITLE
Add validation when creating a list of books

### ISSUE THREAD

**Issue by BrunoHenrique00** [internetarchive/openlibrary#7901]
### Evidence / Screenshot (if possible)
![image](https://github.com/internetarchive/openlibrary/assets/63210374/6915ad67-1e21-41c2-ac1e-70cf57d4b4c1)

### Relevant url?
```
/people/openlibrary/lists
```

### Steps to Reproduce

1. Create a list with a title with more than 255 character, for example
2. Then go to your lists view

* Actual: List title breaks the page style
* Expected: Title to be less than 255 char or front-end not breaking the page

### Details

- **Logged in (Y/N)?** Y
- **Browser type/version?** Chrome Version 112.0.5615.165 (Official Build) (64-bit)
- **Operating system?** Ubuntu
- **Environment (prod/dev/local)?** dev/local

### Proposal & Constraints
Check the title length on both back end and front end to prevent this type of input from the user

**Comment by seabelis** [comment]
I'm not sure this is really an issue. If a patron wants a long name for a list, why should we prevent it?

**Comment by cdrini** [comment]
Agreed with Lisa here. Note this is also only because there is no space in the list name. @davidscotson recently made some changes that should fix this by forcing it to wrap for work titles. Does your fix apply here @davidscotson ?

**Comment by Abhishektharu** [comment]
the changes should be done in page-user.css but I didn't find it. How do I get it.
